src/core/i18n/utils.js

```diff
@@ -234,22 +234,36 @@ type I18nConfig = {|
   _momentDefineLocale?: Function,
 |};
 
+type makeI18nOptions = {|
+  // FIXME this should be tied back to the Intl type.
+  _Intl?: any,
```

Comment:
This was the hack I used to pass the flow checking so any ideas about how to get this to work properly would be much appreciated.